### DISCUSSION TITLE
Unable to package successfully, continuously in build without any errors

### DISCUSSION THREAD

**Discussion by zdd88** [codemagic-ci-cd/codemagic-docs#2452] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

Unable to package successfully, constantly in build without any errors, I don't know where the problem is

### Expected results

1

### Actual results

1

### Build id (optional)

660d2bde929f0d8ce8a6af99

**Comment by zdd88** [comment]
<img width="826" alt="截屏2024-04-03 18 22 49" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/164136096/a0206cd6-8495-4338-89b6-d5b23dd1dc86">
错误截图

**Reply by himesh-cm** [reply]
Hi @zdd88, I see that you were able to resolve the issue. Please don't hesitate to let us know if you still need help with your query.

**Reply by himesh-cm** [reply]
Please note that you should contact our team through the chat widget in the UI for a quicker response, as your team is on a paid plan.

**Comment by zdd88** [comment]
Thanks,it is resolved

---- Replied Message ----
| From | Himesh ***@***.***> |
| Date | 04/08/2024 18:06 |
| To | codemagic-ci-cd/codemagic-docs ***@***.***> |
| Cc | zdd88 ***@***.***>,
Mention ***@***.***> |
| Subject | Re: [codemagic-ci-cd/codemagic-docs] Unable to package successfully, continuously in build without any errors (Discussion #2452) |

Please note that you should contact our team through the chat widget in the UI for a quicker response, as your team is on a paid plan.
